docs/Ecosystem.md

```diff
@@ -79,6 +79,7 @@ Plugins maintained by the fastify team are listed under [Core](#core) while plug
 - [`fastify-google-cloud-storage`](https://github.com/carlozamagni/fastify-google-cloud-storage) Fastify plugin that exposes a GCP Cloud Storage client instance.
 - [`fastify-gql`](https://github.com/mcollina/fastify-gql) A GraphQL server implementation for Fastify with caching and [`graphql-jit`](https://github.com/ruiaraujo/graphql-jit).
 - [`fastify-graceful-shutdown`](https://github.com/hemerajs/fastify-graceful-shutdown) Shutdown Fastify gracefully and asynchronously.
+- [`fastify-gcloud-trace`](https://https://github.com/mkinoshi/fastify-gcloud-trace) Google Cloud Trace API Connector for Fastify.
```

Comment:
This should be in alphabetical order. So, it should be `fastify-gcloud-trace` followed by `fastify-google-cloud-storage`.

Reply:
Oops my bad, I updated it